### PR TITLE
Session recording fix

### PR DESCRIPTION
**Pull request by sabbaka**
Please, do not merge it yet. I will add more tomorrow.

### CODE REVIEW

**Comment by sabbaka** [comment]
I need some to review it once again.

**Comment by dahaic** [comment]
Hi @sabbaka, several observations:
* at least once, "@essionrecording" is present (typo, missing 's')
* "In next step we need to restart nginx server" and you have 'vi' command there
* Installation of packages can be pre-made, so attendee does not have to spend time doing that
* Users might be present on the machine already, to save more time
* sssd-session-recording.conf file can also be premade, with lab only reading it (cat) and copying it to the right place
* have you considered to recommend `nano` instead of `vi`? It's closer to the standard editors that are used.

File: 2019Labs/RHELSecurityLab/documentation/lab12_SessionRecording.adoc [hunk 1]
```diff
@@ -93,18 +125,20 @@ Now you should configure users to be recorded using of ot the following three me
  * sssd config in terminal
```
Comment (dahaic) [inline 2019Labs/RHELSecurityLab/documentation/lab12_SessionRecording.adoc]: This does not newline properly

File: 2019Labs/RHELSecurityLab/documentation/lab12_SessionRecording.adoc [hunk 2]
```diff
@@ -145,9 +179,9 @@ Then you need to restart SSSD, so that changes will take place:
 
     [root@essionrecording]# systemctl restart sssd
     
-Jump directly to "In Practise" section, skip the other user configuration methods.
+Jump directly to "Using Session Player in Cockpit" section, skip the other user configuration methods.
 
-==== One more way to enable recording by changing user's shell and avoiding usage of SSSD (OPTIONAL method)
+==== Lab 12.4.1 One more way to enable recording by changing user's shell and avoiding usage of SSSD (OPTIONAL method)
```
Comment (dahaic) [inline 2019Labs/RHELSecurityLab/documentation/lab12_SessionRecording.adoc]: You are editing jbond in this part, instead of q

**Comment by dahaic** [comment]
OK, I have done the lab in current state and more feedback:
* user `jbond` is not used in the lab at all. Why not get rid of him?
* passwords of users are ... strange, and forces user to copypaste. Can we have something simple, like 'session' for example? 
* when you restart `nginx`, it's not copy-pastable
* at one moment you mention 'restarting cockpit', but it's actually 'restarting nginx'
* not sure if the lab is complete now, but other labs has link to [top] and to [table of contents] at the bottom.

**Pull request merged**